packages/components/button/src/ButtonGroup/README.mdx

```diff
@@ -5,61 +5,44 @@ status: 'stable'
 section: 'buttonComponents'
 slug: /components/button-group/
 github: 'https://github.com/contentful/forma-36/tree/forma-v4/packages/components/button/src/ButtonGroup'
-storybook: 'https://v4-f36-storybook.netlify.app/?path=/story/components-button-buttongroup--basic'
 typescript: ./ButtonGroup.tsx
 ---
 
-import { Props } from '@contentful/f36-docs-utils';
-import { Button } from '@contentful/f36-button';
-import { ChevronDownIcon, PlusIcon } from '@contentful/f36-icons';
+`ButtonGroup` should be used to group buttons whose actions are related, with an option to collapse them together or split them with an equal amount of free space.
 
-## Code examples
+## Import
 
-Example of usage:
+```js static=true
+import { ButtonGroup } from '@contentful/f36-components';
+```
 
-```jsx
-// import { ButtonGroup, Button } from '@contentful/f36-button';
+## Examples
 
-<ButtonGroup testId="group-button">
-  <Button variant="secondary">Button</Button>
-  <Button variant="secondary">Button</Button>
-  <Button
-    variant="positive"
-    aria-label="Open dropdown"
-    startIcon={<ChevronDownIcon />}
-  />
-</ButtonGroup>
-```
+### Collapsed
 
-It's possible to add a divider for buttons that don't have border:
+The most common example is to group primary button with a secondary button or with a button that opens a dropdown with secondary actions.
```

Comment:
hmm, we say here that it is a common case to group primary and secondary button, but we don't show it in the example. Is it really that common btw?

Reply:
I'm going to rephrase it to `One of the common use cases...` and we actually show this in the example below, aren't we?